spec/unit/puppet_x/puppetlabs/strings/yard/puppet_4x_function_handler_spec.rb

```diff
@@ -71,4 +71,74 @@ def max(num_a, num_b)
         RUBY
       }.to output("#{expected_output_not_a_param}\n#{expected_output_also_not_a_param}\n").to_stdout_from_any_process
   end
+
+
+  it "should not issue a warning if the parameter names do  match the docstring" do
```

Comment:
Better to phrase test "names" as a heading for an example rather than expressing a requirements ("should") - e.g. "does not issue a warning when parameter names match", or "no warning is issued when parameter names match".